lib/Controller/AttachmentController.php

```diff
@@ -190,19 +201,21 @@ private function getUploadedFile(string $key): array {
 	 * @param string $sessionToken
 	 * @param string $imageFileName
 	 * @param string|null $shareToken
+	 * @param int $preferRawImage
```

Comment:
I suggest that we use bool here for `prefereRawImage` as you did for the other methods.

Reply:
I don't trust passing boolean values in GET params. But maybe it's fine, do you actually do it somewhere else? Do you pass the "true" or "false" string and the middleware converts it to a boolean?

I would still prefer keeping it like that and passing 1 for true and 0 for false :grin:.